### ISSUE TITLE
[New Feature] Switch to use Ansible-core 2.12 after it's release

### ISSUE THREAD

**Issue by Tomorrow9** [vmware/ansible-vsphere-gos-validation#136]
##### SUMMARY

Ansible-core 2.12 release schedule:
```
    2021-09-24 Feature Freeze No new functionality (including modules/plugins) to any code

    2021-09-27 Beta 1

    2021-10-04 Beta 2 (if necessary)

    2021-10-18 Release Candidate 1 (and stable-2.12 branching from devel)

    2021-10-25 Release Candidate 2 (if necessary)

    2021-11-08 Release
```

##### ISSUE TYPE
- Feature Idea

##### ENVIRONMENT / OS

All

**Comment by Tomorrow9** [comment]
changelog of Ansible core 2.12:
https://github.com/ansible/ansible/blob/stable-2.12/changelogs/CHANGELOG-v2.12.rst

**Comment by Tomorrow9** [comment]
Ansible issue about `end_play`:
https://github.com/ansible/ansible/issues/76672

Our fix about this issue merged:
https://github.com/vmware/ansible-vsphere-gos-validation/pull/213

**Comment by keirazhang** [comment]
https://github.com/vmware/ansible-vsphere-gos-validation/milestone/2 has released with ansible-core 2.12.1. See https://github.com/vmware/ansible-vsphere-gos-validation/releases/tag/v1.3.
